test/server/lib/filenamesUtils.ts

```diff
@@ -0,0 +1,10 @@
+import { filenameContentDisposition } from 'app/server/lib/filenamesUtils';
+import {assert} from 'chai';
+
+describe('filenamesUtils', function() {
```

Comment:
This would be more convincing with more tests, and some evidence that the encodings will be parsed as expected.